CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Changes
+- Fixed 'Raise helpful error if the same needs is not unique[#531](https://github.com/luckyframework/lucky/issues/531)
```

Comment:
Don't worry about adding in changelog entries. These get added in all at the same time.

Reply:
Done